.github/workflows/build.yml

```diff
@@ -32,9 +33,9 @@ jobs:
         with:
           path: |
             /home/runner/.ccache
-          key: ${{ runner.os }}-${{ matrix.toolchain }}-${{ matrix.protocol }}-cacheID-${{ steps.cache_extra_id.outputs.id }}
+          key: ${{ runner.os }}-${{ matrix.toolchain }}-cacheID-${{ steps.cache_extra_id.outputs.id }}
```

Comment:
maybe add a comment here that explains why we can omit `protocol` (very few files need to be recompiled) for future us that will probably forget about this

Reply:
Sure, fixed.